src/wr_logic_ai/src/wr_logic_ai/coordinate_manager.py

```diff
@@ -11,27 +11,24 @@ def __init__(self) -> None:
 
     @staticmethod
     def get_coordinate():
-        print(CoordinateManager.ptr)
         return CoordinateManager.coordinates[CoordinateManager.ptr]
         
     @staticmethod
     def previous_line():
```

Comment:
Better name: `previous_coordinate`

Reply:
Addressed in [277b00a](https://github.com/WisconsinRobotics/WRover_Software/pull/48/commits/277b00a47c8537aba6ba2aeeb6c09d9b5ea4d3eb).